src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -778,6 +769,19 @@ impl Node {
 
         }
 
+
+        // === Action Bar ===
+
+        frp::extend! { network
+            out.source.skip                    <+ action_bar.action_skip;
+            out.source.freeze                  <+ action_bar.action_freeze;
+            frp.show_quick_action_bar_on_hover <+ no_error_set;
```

Comment:
Moving icons is bad solution IMO. The eye icon can be grayed out instead. Changing controls placement is super confusing to users. Peopel will get used to the fact that first icon is visualization. Imagine a situation when you try to click it, but then error occurs and you click other icon because it just changed place - this is a very bad UX

Reply:
@wdanilo Then we need to come up with a proper visual design for this. Fading out would probably not work because it would be confusable with the current active/not-active semantics, unless we re-design those.